src/build/tasks/postcss/__tests__/increase-specifity.test.ts

```diff
@@ -1,17 +1,16 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+import { test, expect } from 'vitest';
 import postcss from 'postcss';
 import postCSSIncreaseSpecificity from '../increase-specifity';
 
-function setupTest(
-  test: (process: (input: string) => Promise<string>) => Promise<void>
-): (callback: () => void) => void {
-  return (callback: () => void) => {
```

Comment:
This test used `test('...', done => {})` syntax, which is not supported by vitest, had to rewrite